actors/power/tests/power_actor_tests.rs

```diff
@@ -1186,8 +1196,129 @@ mod cron_batch_proof_verifies_tests {
+            sector_nums: vec![infos[0].sector_id.number, infos[2].sector_id.number],
+        };
+
+        expect_query_network_info(&mut rt, &h);
+
+        let state: State = rt.get_state();
+
+        // expect sends for confirmed sectors
+        let params = ConfirmSectorProofsParams {
+            sectors: cs.sector_nums,
+            reward_smoothed: h.this_epoch_reward_smoothed.clone(),
+            reward_baseline_power: h.this_epoch_baseline_power().clone(),
+            quality_adj_power_smoothed: state.this_epoch_qa_power_smoothed,
+        };
+
+        rt.expect_send(
+            cs.miner,
+            CONFIRM_SECTOR_PROOFS_VALID_METHOD,
+            RawBytes::serialize(params).unwrap(),
+            TokenAmount::from(0u8),
+            RawBytes::default(),
+            ExitCode::OK,
+        );
+
+        rt.expect_batch_verify_seals(infos, res);
+
+        // expect power sends to reward actor
+        rt.expect_send(
+            *REWARD_ACTOR_ADDR,
+            UPDATE_NETWORK_KPI,
+            RawBytes::serialize(BigIntSer(&BigInt::zero())).unwrap(),
+            TokenAmount::from(0u8),
+            RawBytes::default(),
+            ExitCode::OK,
+        );
+
+        rt.expect_validate_caller_addr(vec![*CRON_ACTOR_ADDR]);
+
+        rt.set_epoch(0);
+        rt.set_caller(*CRON_ACTOR_CODE_ID, *CRON_ACTOR_ADDR);
+
+        rt.call::<PowerActor>(Method::OnEpochTickEnd as u64, &RawBytes::default()).unwrap();
+
+        rt.verify();
+    }
+
+    #[test]
+    fn cron_tick_does_not_fail_if_batch_verify_seals_fails() {
+        let (mut h, mut rt) = setup();
+        h.create_miner_basic(&mut rt, OWNER, OWNER, MINER_1).unwrap();
+
+        let infos: Vec<_> = (1..=3).map(create_basic_seal_info).collect();
+        infos.iter().for_each(|info| {
+            h.submit_porep_for_bulk_verify(&mut rt, MINER_1, info.clone()).unwrap()
+        });
+
+        let cs = ConfirmedSectorSend {
+            miner: MINER_1,
+            sector_nums: vec![
+                infos[0].sector_id.number,
+                infos[1].sector_id.number,
+                infos[2].sector_id.number,
+            ],
+        };
+
+        expect_query_network_info(&mut rt, &h);
+
+        let state: State = rt.get_state();
+
+        // expect sends for confirmed sectors
+        let params = ConfirmSectorProofsParams {
+            sectors: cs.sector_nums,
+            reward_smoothed: h.this_epoch_reward_smoothed.clone(),
+            reward_baseline_power: h.this_epoch_baseline_power().clone(),
+            quality_adj_power_smoothed: state.this_epoch_qa_power_smoothed,
+        };
+
+        rt.expect_send(
+            cs.miner,
+            CONFIRM_SECTOR_PROOFS_VALID_METHOD,
+            RawBytes::serialize(params).unwrap(),
+            TokenAmount::from(0u8),
+            RawBytes::default(),
+            ExitCode::OK,
+        );
+
+        rt.expect_batch_verify_seals(infos.clone(), Ok(batch_verify_default_output(&infos)));
```

Comment:
The go test of the same name does not seem to have the send above.  I think the reason for the discrepancy is that you are not expecting a failure here.  If you look closely the go test at line 1107 expects a failure from the sys call 
```golang
		rt.ExpectBatchVerifySeals(infos, batchVerifyDefaultOutput(infos), fmt.Errorf("fail"))
```

The go mock runtime is a bit confusing because it specifies both a return and an error value.  The rust mock runtime does not yet appear to have the capability to mock out a failed sys call.  To complete this PR you'll need to fix this.  Once you do this the power actor will get a failure from the batch verify sys call and you won't need to add this extra expect send to get the test to pass, matching the go test.